README.md

```diff
@@ -50,7 +54,19 @@ A DP project is run for each batch and the locations and images for each DP run
 The larger the batch size, the faster a stream will complete (DP will not need to be initialized as often).
 However, a larger batch size also corresponds to more intermediate files and more memory being allocated at once by DP (step 3) and PyCytominer (step 4).
 
-Once a stream is set up, it processes each batch as follows:
+**For CellProfiler Project:**
+
+4) Initialize cp_metadata.
+The CellProfiler metadata compiler needs to be initialized as to convert the `data_to_process.tsv` into a `.csv` file for CellProfiler to intake the metadata.
```

Comment:
I think this more accurately encompasses what is demonstrated in `example_cp.ipynb` but correct me if I'm wrong.
```suggestion
`idrstream.CellProfilerRun().convert_tsv_to_csv()` needs to be used to initialize metadata for CellProfiler use. Check [example_cp.ipynb](example_notebooks/example_cp.ipynb) for more info.
```

Reply:
:+1: simpler and to the point! Added.